app/http/controllers/user2.js

```diff
@@ -52,7 +52,7 @@ module.exports.createUser = function (User) {
       prefLocale: req.body.user.prefLocale,
       referrer: req.body.user.referrer,
       lastLoggedIn: new Date(),
-      teach: req.body.teach
+      client_id: req.body.oauth ? req.body.oauth.client_id : ""
```

Comment:
Actually, revert this change since createUser still needs the teach flag, and put the `client_id` attribute in https://github.com/mozilla/login.webmaker.org/blob/d1cc58b558d11d819ac407bae68670094c05be0d/app/db/models/index.js#L186